frontend/src/stores/useProjectStore.ts

```diff
@@ -24,11 +26,28 @@ import {
   DEFAULT_PROJECT_COLOR
 } from '../config/projects'
 
+import { PASTE_POSITION_OFFSET } from 'frontend/src/config/rendering.js'
+
 /**
  * A stored project has an extra `_id` field which is used to tell identify it
  */
 export type StoredProject = Project & {_id: string}
 
+export enum InsertGroupResponseType {
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Weird you need to disable, but I think I know the problem and will fix myself in a seperate PR :+1:

Reply:
Oh yes I meant to single this out in a comment, linter wasn't liking enums 😢 Thank you, that would be great!